### ISSUE TITLE
ModuleNotFoundError occurs when running after installing with pip.

### ISSUE THREAD

**Issue by soimsoim** [fossology/atarashi#70]
I tried running it after installing it with pip on python 3.6.9. Then, the following error occurs.
Are there additional modules I need to install?

```
$pip install atarashi 
$atarashi -h
Traceback (most recent call last):
  File "/home/soimkim/test/venv/bin/atarashi", line 5, in <module>
    from atarashi.atarashii import main
  File "/home/soimkim/test/venv/lib/python3.6/site-packages/atarashi/atarashii.py", line 26, in <module>
    from atarashi.agents.cosineSimNgram import NgramAgent
  File "/home/soimkim/test/venv/lib/python3.6/site-packages/atarashi/agents/cosineSimNgram.py", line 30, in <module>
    from atarashi.agents.atarashiAgent import AtarashiAgent
  File "/home/soimkim/test/venv/lib/python3.6/site-packages/atarashi/agents/atarashiAgent.py", line 27, in <module>
    from atarashi.libs.commentPreprocessor import CommentPreprocessor
  File "/home/soimkim/test/venv/lib/python3.6/site-packages/atarashi/libs/commentPreprocessor.py", line 23, in <module>
    import code_comment  # https://github.com/amanjain97/code_comment/
ModuleNotFoundError: No module named 'code_comment'
```

**Comment by GMishx** [comment]
Thanks for writing in, we changed the dependency some time back but didn't push new version of atarashi on PyPi.

**Comment by GMishx** [comment]
Hello @soimsoim , we have pushed the version to PyPI: https://pypi.org/project/atarashi/0.0.11

Just update the package and it should work now.

**Comment by GMishx** [comment]
Closing the issue as the new version was released in #71

Please reopen if the issue persists.
